docs/architecture/event-flows-proposal.md

```diff
@@ -0,0 +1,85 @@
+# Eventing flow proposal
+
+## Overview
+
+This document shows a proposal of two ways of eventing flow we want to support with use of the [Management Plane](./terminology.md#management-plane)
+
+## Eventing with Kyma internal Event Bus
+
+![Management Plane Components](assets/internal-event-flow.svg)
+
+1. The event is registered using **Management Plane** API.
+2. The **Director** sends an event url to Application.
+3. The **Agent** fetches the event data and passes it to Kyma Event Bus through Event Gateway. It will support multiple protocols like HTTP, MQTT along with multiple authentication (e.g. OpenID Connect) and authorization (e.g. OAuth2) methods.  
+4. The Application publishes an event.
```

Comment:
that sentence: "It will support multiple protocols like HTTP, MQTT along with multiple authentication (e.g. OpenID Connect) and authorization (e.g. OAuth2) methods." That's a feature of EventGateway. And Application during eventAPI registration could pick up one of those options they need.

Reply:
refactored the whole section